tools/clang/lib/SPIRV/SpirvEmitter.h

```diff
@@ -767,42 +768,6 @@ class SpirvEmitter : public ASTConsumer {
                                           SourceLocation,
                                           SourceRange range = {});
 
-private:
-  /// Translates the given frontend APValue into its SPIR-V equivalent for the
```

Comment:
Shall those comments also be moved to the header? Especially for function like `translateAsFloat` where we allow casting to some extent.

Reply:
Ah, yes, they should have been moved. I've fixed it.